docs/deployment/docker.md

```diff
@@ -213,7 +214,11 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     apt-get install -y --no-install-recommends libsqlite3-dev && \
     rm -rf /var/lib/apt/lists/*
 
+RUN corepack enable && npm install -g corepack@0.27.0 && corepack install --global yarn@1.22.22
+RUN mkdir -p /home/node/.cache/node/corepack/v1 && chown -R node:node /home/node
+
 USER node
+RUN git config --global http.sslverify false
```

Comment:
Following my previous comment about `git` this would not be needed, feel free to add it to your Dockerfile